src/app/clusters/basic-information/BasicInformationCluster.cpp

```diff
@@ -247,175 +254,194 @@ inline CHIP_ERROR ReadProductAppearance(DeviceInstanceInfoProvider * deviceInfoP
         return ReadConfigurationString(deviceInfoProvider, &DeviceInstanceInfoProvider::GetHardwareVersionString,
-                                       false /* unimplementedAllowed */, aEncoder);
+                                       false /* unimplementedAllowed */, encoder);
     case SoftwareVersion::Id:
-        return ReadSoftwareVersion(configManager, aEncoder);
+        return ReadSoftwareVersion(configManager, encoder);
     case SoftwareVersionString::Id:
-        return ReadSoftwareVersionString(configManager, aEncoder);
+        return ReadSoftwareVersionString(configManager, encoder);
     case ManufacturingDate::Id:
-        return ReadManufacturingDate(deviceInfoProvider, aEncoder);
+        return ReadManufacturingDate(deviceInfoProvider, encoder);
     case PartNumber::Id:
         return ReadConfigurationString(deviceInfoProvider, &DeviceInstanceInfoProvider::GetPartNumber,
-                                       true /* unimplementedAllowed */, aEncoder);
+                                       true /* unimplementedAllowed */, encoder);
     case ProductURL::Id:
         return ReadConfigurationString(deviceInfoProvider, &DeviceInstanceInfoProvider::GetProductURL,
-                                       true /* unimplementedAllowed */, aEncoder);
+                                       true /* unimplementedAllowed */, encoder);
     case ProductLabel::Id:
         return ReadConfigurationString(deviceInfoProvider, &DeviceInstanceInfoProvider::GetProductLabel,
-                                       true /* unimplementedAllowed */, aEncoder);
+                                       true /* unimplementedAllowed */, encoder);
     case SerialNumber::Id:
         return ReadConfigurationString(deviceInfoProvider, &DeviceInstanceInfoProvider::GetSerialNumber,
-                                       true /* unimplementedAllowed */, aEncoder);
+                                       true /* unimplementedAllowed */, encoder);
     case UniqueID::Id:
-        return ReadUniqueID(configManager, aEncoder);
+        return ReadUniqueID(configManager, encoder);
     case CapabilityMinima::Id:
-        return ReadCapabilityMinima(aEncoder);
+        return ReadCapabilityMinima(encoder);
     case ProductAppearance::Id:
-        return ReadProductAppearance(deviceInfoProvider, aEncoder);
+        return ReadProductAppearance(deviceInfoProvider, encoder);
     case SpecificationVersion::Id:
-        return aEncoder.Encode(Revision::kSpecificationVersion);
+        return encoder.Encode(Revision::kSpecificationVersion);
     case MaxPathsPerInvoke::Id:
-        return aEncoder.Encode<uint16_t>(CHIP_CONFIG_MAX_PATHS_PER_INVOKE);
+        return encoder.Encode<uint16_t>(CHIP_CONFIG_MAX_PATHS_PER_INVOKE);
     case ConfigurationVersion::Id:
-        return ReadConfigurationVersion(configManager, aEncoder);
+        return ReadConfigurationVersion(configManager, encoder);
+    case Reachable::Id:
+        return encoder.Encode<bool>(true);
```

Comment:
Why is the `<bool>` needed?  Shouldn't be needed...  `true` is most certainly a bool, so that should end up with the right template instantiation anyway, no?